libs/libyang/Makefile

```diff
@@ -55,20 +55,17 @@ define Package/libyang/description
 endef
 
 CMAKE_OPTIONS += \
-	-DENABLE_LYD_PRIV:BOOL=ON \
-	-DCMAKE_BUILD_TYPE:String="Release" \
-	-DGEN_LANGUAGE_BINDINGS=ON \
-	-DGEN_PYTHON_BINDINGS=OFF
+	-DCMAKE_BUILD_TYPE:String="Release"
 
 define Package/libyang/install
 	$(INSTALL_DIR) $(1)/usr/lib
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libyang.so* $(1)/usr/lib/
 
-	$(INSTALL_DIR) $(1)/usr/lib/libyang1
-	$(INSTALL_DIR) $(1)/usr/lib/libyang1/extensions
-	$(INSTALL_DIR) $(1)/usr/lib/libyang1/user_types
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/lib/libyang1/extensions/* $(1)/usr/lib/libyang1/extensions
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/lib/libyang1/user_types/* $(1)/usr/lib/libyang1/user_types
+#	$(INSTALL_DIR) $(1)/usr/lib/libyang1
+#	$(INSTALL_DIR) $(1)/usr/lib/libyang1/extensions
+#	$(INSTALL_DIR) $(1)/usr/lib/libyang1/user_types
+#	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/lib/libyang1/extensions/* $(1)/usr/lib/libyang1/extensions
+#	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/lib/libyang1/user_types/* $(1)/usr/lib/libyang1/user_types
```

Comment:
can we remove these lines entinery instead of commenting them out?

Reply:
yes, I'll do it in few minutes